samples/bluetooth/unicast_audio_server/prj.conf

```diff
@@ -0,0 +1,7 @@
+CONFIG_BT=y
+CONFIG_BT_DEBUG_LOG=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_AUDIO=y
+CONFIG_BT_AUDIO_UNICAST_SERVER=y
+# TODO: Remove, but is currently needed as UNICAST_SERVER cannot build on its own
```

Comment:
Taking this place, I think we should define the four roles of Broadcast in accordance with the BAP specification.

Reply:
Aren't they already defined now? 
We have UNICAST_SERVER, UNICAST_CLIENT, BROADCAST_SOURCE and BROADCAST_SINK. The only roles we are missing are the BROADCAST_ASSISTANT and SCAN_DELEGATOR.